src/win/pipe.c

```diff
@@ -1226,154 +1227,111 @@ static void uv_queue_non_overlapped_write(uv_pipe_t* handle) {
 }
 
 
-static int uv_pipe_write_impl(uv_loop_t* loop,
-                              uv_write_t* req,
-                              uv_pipe_t* handle,
-                              const uv_buf_t bufs[],
-                              unsigned int nbufs,
-                              uv_stream_t* send_handle,
-                              uv_write_cb cb) {
-  int err;
-  int result;
-  uv_tcp_t* tcp_send_handle;
-  uv_write_t* ipc_header_req = NULL;
-  uv_ipc_frame_uv_stream ipc_frame;
+static int uv__build_coalesced_write_req(uv_write_t* user_req,
```

Comment:
I think we should mention this in the docs, that the buffers will get combined

Reply:
I'm not sure I agree, as this entirely transparent (and likely unnoticeable) to the user.
For now, created an issue for it: https://github.com/libuv/libuv/issues/1855